instrumentation/spring/spring-webmvc-3.1/spring-webmvc-3.1-auto/src/test/groovy/test/filter/FilteredAppConfig.groovy

```diff
@@ -16,6 +16,8 @@
 
 package test.filter
 
+import org.springframework.boot.web.embedded.tomcat.TomcatConnectorCustomizer
```

Comment:
Make sure that you have [correct import settings](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/contributing/intellij-setup.md)

Reply:
Yep, done.